### PR TITLE
removing redundant state updates

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

File: src/actions/publishablePositions.js [hunk 1]
```diff
@@ -51,12 +51,7 @@ export function publishablePositionsFetchData(query = {}) {
         });
       })
       .catch((err) => {
-        if (err?.message === 'cancel') {
-          batch(() => {
-            dispatch(publishablePositionsLoading(true));
-            dispatch(publishablePositionsErrored(false));
```
Comment (elizabeth-jimenez) [inline src/actions/publishablePositions.js:57]: was redundant with lines 39 and 40

File: src/actions/publishablePositions.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ export function publishablePositionsSuccess(results) {
 }
 export function publishablePositionsFetchData(query = {}) {
   return (dispatch) => {
-    if (cancelPPData) { cancelPPData('cancel'); dispatch(publishablePositionsLoading(true)); }
```
Comment (elizabeth-jimenez) [inline src/actions/publishablePositions.js:37]: loading not needed cause L39 and 40 cause a cancel re-calls this whole function

File: src/actions/publishablePositions.js (hunk 1 above)
Comment (M00staff) [reply src/actions/publishablePositions.js:57]: i think i follow - so what happens when the call is canceled?

wouldn't we still need an else statement to at least flip the loading action to false?  Or would the timed out calls just keep spinning?

Comment (elizabeth-jimenez) [reply src/actions/publishablePositions.js:57]: when the call is cancelled publishablePositionsFetchData gets called again - or at least starts over(not sure of the nuance there).


we don't really want to flip the loading to false on a cancel, cause it really is just saying "go again, even tho u havent finished the last one", so it's almost an overlapping of calls, with no break in their completion state.


timed out calls are legit errors, so those will get handled on L55

**Review by M00staff — APPROVED** [review]
👍

**Pull request merged**